[2023-06-13T16:17:01.292+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:39.273378+00:00 [queued]>
[2023-06-13T16:17:01.305+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:39.273378+00:00 [queued]>
[2023-06-13T16:17:01.306+0200] {taskinstance.py:1308} INFO - Starting attempt 2 of 1
[2023-06-13T16:17:01.328+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): task_extract> on 2023-06-13 14:00:39.273378+00:00
[2023-06-13T16:17:01.338+0200] {standard_task_runner.py:57} INFO - Started process 10038 to run task
[2023-06-13T16:17:01.348+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'airflow_use_case', 'task_extract', 'manual__2023-06-13T14:00:39.273378+00:00', '--job-id', '48', '--raw', '--subdir', '/Users/simba/Documents/Converteo/cas_airflow1/dags/dag_exemple.py', '--cfg-path', '/var/folders/yj/q_cm8stj3td4yz26dm7d78j80000gn/T/tmpjlcykb2k']
[2023-06-13T16:17:01.354+0200] {standard_task_runner.py:85} INFO - Job 48: Subtask task_extract
[2023-06-13T16:17:01.434+0200] {task_command.py:410} INFO - Running <TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:39.273378+00:00 [running]> on host mbp-de-admin-1.home
[2023-06-13T16:17:01.538+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='airflow_use_case' AIRFLOW_CTX_TASK_ID='task_extract' AIRFLOW_CTX_EXECUTION_DATE='2023-06-13T14:00:39.273378+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-13T14:00:39.273378+00:00'
[2023-06-13T16:30:53.033+0200] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:39.273378+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2023-06-13T16:30:53.034+0200] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:39.273378+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2023-06-13T16:30:53.035+0200] {local_task_job_runner.py:154} INFO - Task is not able to be run
[2023-06-13T16:31:28.208+0200] {local_task_job_runner.py:291} WARNING - State of this instance has been externally set to None. Terminating instance.
[2023-06-13T16:31:28.294+0200] {process_utils.py:131} INFO - Sending 15 to group 10038. PIDs of all processes in the group: [10038]
[2023-06-13T16:31:28.295+0200] {process_utils.py:86} INFO - Sending the signal 15 to group 10038
[2023-06-13T16:32:28.295+0200] {process_utils.py:149} WARNING - process psutil.Process(pid=10038, name='Python', status='running', started='16:17:01') did not respond to SIGTERM. Trying SIGKILL
[2023-06-13T16:32:28.297+0200] {process_utils.py:86} INFO - Sending the signal 9 to group 10038
[2023-06-13T16:32:28.312+0200] {process_utils.py:79} INFO - Process psutil.Process(pid=10038, name='Python', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='16:17:01') (10038) terminated with exit code -9
[2023-06-13T16:32:28.313+0200] {standard_task_runner.py:172} ERROR - Job 48 was killed before it finished (likely due to running out of memory)
